core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -97,7 +97,7 @@
  * @param <ReqT> type of message sent one or more times to the server.
  * @param <RespT> type of message received one or more times from the server.
  */
-@DoNotMock("Use InProcessTransport and make a fake server instead")
+@DoNotMock("Use InProcessServerBuilder and make a full featured in process server instead")
```

Comment:
"full featured" isn't precise either. The `InProcessServer` does not feature load-balancing for example. Can we just say `Use InProcessServerBuilder and stub an in-process server instead`?